packages/edit-site/src/components/style-book/color-examples.tsx

```diff
@@ -18,20 +18,21 @@ import {
  */
 import type { Color, Gradient } from './types';
 
-const ColorExamples = ( { colors, type } ): JSX.Element | null => {
+const ColorExamples = ( { colors, type, columns = 2 } ): JSX.Element | null => {
 	if ( ! colors ) {
 		return null;
 	}
 
 	return (
-		<Grid columns={ 2 } rowGap={ 8 } columnGap={ 16 }>
+		<Grid columns={ columns } rowGap={ 8 } columnGap={ 16 }>
 			{ colors.map( ( color: Color | Gradient ) => {
```

Comment:
Do we always want it to show 4 cols, regardless of width? This is how it looks on small screens:

<img width="298" alt="Screenshot 2024-12-05 at 1 45 42 pm" src="https://github.com/user-attachments/assets/313dc367-b188-47b9-bc23-d44dbc85334d">

If we wanted to make it responsive, we could instead of `columns` use `templateColumns` with something like `repeat(auto-fill, minmax(150px, 1fr))`

It's not too bad as it is though so I don't feel strongly about it.

Reply:
It would be nice to fall back to 2 columns on mobile, 👍 👍

Reply:
:+1: ok, thanks for the suggestion. I added that in the latest commits

Reply:
Thanks, LGTM!